.circleci/config.yml

```diff
@@ -819,7 +819,7 @@ jobs:
           name: Install test dependencies
           command: |
             apt-get update
-            apt-get install -qqy --no-install-recommends nodejs npm cvc4
+            apt-get install -qqy --no-install-recommends nodejs npm cvc4 z3
```

Comment:
With the smt queries fixed, we can also run it via the solc-js callback by having a solver installed in the system. However, cvc4 does not have a Horn solver so the test is kinda null at the end. I installed z3 so that the solc-js smt callback tests can also run a Horn solver locally. This is still not enough because this installs z3 4.4 cus... Debian. Still have to find a solution to install the same z3 as our tests here